packages/server/manage_container/client/manage_to_db/client.go

```diff
@@ -326,3 +328,19 @@ func (c Client) GetMatchingColumn(dataID string) (int32, error) {
 
 	return data.Meta.MatchingColumn, nil
 }
+
+func (c Client) CreateStatusFile(jobUUID string) {
+	ls.Lock(jobUUID)
+	defer ls.Unlock(jobUUID)
+	path := fmt.Sprintf("%s/%s", resultDbPath, jobUUID)
+	os.Mkdir(path, 0777)
+	fp, _ := os.Create(fmt.Sprintf("%s/status_RECEIVED", path))
```

Comment:
Please refer to CC's code.

https://github.com/acompany-develop/QuickMPC/blob/985df54425ca54289e762e9507d47ec62463f388/packages/server/computation_container/client/computation_to_db/client.cpp#L224-L237

Reply:
[61fa5ec](https://github.com/acompany-develop/QuickMPC/pull/150/commits/61fa5ec88b5f0d5bca5322f53f36c4bbab621208)

Reply:
@KotaTakahashi9320  
Thank you.

We can use `pb_types.JobStatus_RECEIVED` instead of magic number.

Reply:
@arukuka 
Thank you!
I fix it with [5600a08](https://github.com/acompany-develop/QuickMPC/pull/150/commits/5600a0850519a31af30d5e7f5e1136bbba5fbc8e)